microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -343,8 +345,8 @@ Users should not be misled to click on advertisements to fully use the extension
 ###### 1.10.2 Policies and agreements
 
 Any ad content that's displayed by your extension must conform to the following:
-* The [Microsoft Advertising Network policies](https://about.ads.microsoft.com/policies/home).
-* The advertising requirements of the [App Developer Agreement](/legal/windows/agreements/app-developer-agreement).
+* [Microsoft Advertising policies and agreements](https://help.ads.microsoft.com/#apex/ads/en/n5095/0).
+* The advertising requirements of the [Microsoft Store App Developer Agreement](https://go.microsoft.com/fwlink/?linkid=528905).
 * The advertising requirements of the developer policies for the Microsoft Edge Add-ons store.
 
 
```

Comment:
urls in line 362 are being redirected, so updated the URLs and link text, from:
```
must comply with 
[Microsoft Creative Specifications Policy]
(https://about.ads.microsoft.com/policies/creative-specs)
and
[Microsoft Advertising Network policies]
(https://about.ads.microsoft.com/policies/home).
```
to:
```
must comply with 
[Creative specs]
(https://help.ads.microsoft.com/#apex/ads/en/ext60207) 
and other 
[Microsoft Advertising policies and agreements]
(https://help.ads.microsoft.com/#apex/ads/en/n5095/0).
```